dataverse-webapp/src/main/webapp/resources/vecler/theme-base.scss

```diff
@@ -1495,6 +1495,19 @@ tr.ui-state-highlight a:not(.btn), .ui-widget-content tr.ui-state-highlight a:no
 	}
 }
 
+/* Scrollable tables */
+
+.ui-datatable:not(.headerless-table) table {
+	min-width: 65em;
```

Comment:
I feel like some of the tables could be displayed in smaller width sizes with no problem - especially those where you have files listing like:
files list on dataset page
files list on create dataset page
files list on edit files metadata page
add file to dataset page
reorder datafiles page
we have a lot of blank space there

Reply:
Files tables still need some min width limit in place - without it, they're unusable on mobile devices. I've made a seperate class `medium-min-width` with `40em` limit instead of `65em`, which should help a bit.
License/file reordering tables had their limit removed - they work fine even in 320px.